.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,78 @@
+name: Bug Report
+description: File a report for a non-blocking bug, i.e., issues that can be worked around.
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: A concise description of what you're experiencing.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        1. With this config...
+        1. Run '...'
+        1. See error...
+    validations:
+      required: false
+  - type: checkboxes
```

Comment:
I would propose that the issues reported in this repo should only be about cbuild2cmake.
Do we really need these check-boxes for all other tools in the CMSIS-Toolbox?

Reply:
[Addressed review comments](https://github.com/Open-CMSIS-Pack/cbuild2cmake/pull/224/commits/71d9d899507ed10f50435ded3a371af5f7779778)